modules/apps/click-to-chat/click-to-chat-web/src/main/resources/META-INF/resources/dynamic_include/hubspot.jsp

```diff
@@ -52,7 +52,7 @@ if (themeDisplay.isSignedIn() && (parts.length > 1)) {
 	<c:when test="<%= themeDisplay.isSignedIn() && (parts.length > 1) %>">
 		<c:choose>
 			<c:when test="<%= Validator.isNull(identificationToken) %>">
-				<aui:script>
+				<aui:script position="inline">
 					Liferay.Util.openToast({
```

Comment:
Won't this cause an issue because it can be loaded before Liferay object is available?

Reply:
I'm basing the changes on the fact that previously, these `<aui:script>` were just `<script>`, so basically this reverts things to the old situation. Just in case, for this one I've checked the git history and I discovered I've missed one, so thanks a lot!

I've force-pushed